core/runtime/runtime_api/impl/offchain_worker_api.cpp

```diff
@@ -7,31 +7,24 @@
 #include "runtime/runtime_api/impl/offchain_worker_api.hpp"
 
 #include "application/app_configuration.hpp"
+#include "log/logger.hpp"
 #include "offchain/impl/runner.hpp"
 #include "offchain/offchain_worker_factory.hpp"
 #include "runtime/executor.hpp"
 
 namespace kagome::runtime {
-  constexpr size_t kMaxThreads = 3;
-  constexpr size_t kMaxTasks = 1000;
-
-  outcome::result<void> callOffchainWorkerApi(
```

Comment:
Keep `callOffchainWorkerApi`, used to benchmark offchain workers.
(or move everything from `OffchainWorkerApiImpl::offchain_worker` to caller)